crates/proof-of-sql/src/base/commitment/column_bounds.rs

```diff
@@ -537,7 +533,7 @@ mod tests {
 
         let timestamp_column = OwnedColumn::<Curve25519Scalar>::TimestampTZ(
             PoSQLTimeUnit::Second,
-            PoSQLTimeZone::UTC,
+            timezone::PoSQLTimeZone::Utc,
```

Comment:
changing "UTC" to "Utc" because thats how chrono represents it

Reply:
Yup. It also looks better in terms of naming convention as well.

Reply:
it kind of blew up the diff sorry about that